homeassistant/helpers/update_coordinator.py

```diff
@@ -65,6 +65,7 @@ def __init__(
         update_interval: timedelta | None = None,
         update_method: Callable[[], Awaitable[_DataT]] | None = None,
         request_refresh_debouncer: Debouncer[Coroutine[Any, Any, None]] | None = None,
+        force_update: bool = True,
```

Comment:
Maybe `always_update` instead?
Especially if it's opt-out it would seem off with `force_update`

Reply:
sounds better to me.  Will adjust 👍